velox/vector/tests/DecodedVectorTest.cpp

```diff
@@ -113,18 +123,37 @@ class DecodedVectorTest : public testing::Test, public VectorTestBase {
   }
 
   template <typename T>
-  void testFlat(vector_size_t cardinality = 10010) {
-    auto cardData = genTestData<T>(cardinality, /* includeNulls */ true);
+  void testFlat(
+      vector_size_t cardinality = 10010,
+      const TypePtr& type = CppToType<T>::create()) {
+    auto cardData = genTestData<T>(
```

Comment:
can we make type the second argument to avoid listing the other defaults?

Reply:
Done.